test/unit/auth/auth-config.spec.ts

```diff
@@ -769,6 +783,17 @@ describe('OIDCConfig', () => {
       expect(config.enabled).to.be.true;
     });
 
+    it('should set readonly property clientSecret', () => {
```

Comment:
Can you add a test for legacy flow (when we didn't expose responseType)?
Basically, what happens if someone was enabling an OIDC provider (without any response type) and upgrades to a new version of the Admin SDK. Will the old code continue to work without any new error?